services/db-manager/src/main/resources/db/changelog/changes/004_create_exchanges_table.yaml

```
@@ -0,0 +1,68 @@
+              - column:
+                  name: exchange_name
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: country_code # ISO 3166-1 alpha-2 country code
+                  type: VARCHAR(2)
+              - column:
+                  name: city
+                  type: VARCHAR(100)
+              - column: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)